.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-name: "search-insights"
+        dependency-type: "production"
```

Comment:
nit: add:

```
commit-message:
  prefix: "chore"
```

Reply:
Missed this message. Fixed here: https://github.com/algolia/search-insights-gtm/pull/50